### DISCUSSION TITLE
Daily timelapse not stopping at set time

### DISCUSSION THREAD

**Discussion by MarvinLo1** [gopro/labs#607] [General]
I've been trying to do a daily timelapse using a set interval with GoPro 10 however I'm having issues with the script not working properly.

I've tried the script: !MTUSB=1!06:00NmTPfWp.60r!S!12600E!1R however the GoPro does not stop after 4 hours and keeps taking images until I stop the camera.

I then tried a sunrise/sunset script: !r-300NmTPp.1800!S!s300E1R and it continues to take photos throughout the night and the camera does not stop.

On the GoPro I have timelapse photo enabled and raw output along with trust usb (MTUSB) since I'm using an external power source.

Any help troubleshooting would be appreciated

**Comment by dnewman-gpsw** [comment]
Battery in or out of the camera?  I don't see an issue with the scripts, but if the battery is out, that can cause issues with the internal clock (even under USB power.)

General note:
Don't combine one time setting like !MTUSB=1  <- use a separate QR code.
With repeated action like !06:00NmTPfWp.60r!S!12600E!1R   (!1R will repeat everything.)

**Comment by MarvinLo1** [comment]
I have been testing it with the battery out but I can try putting it in and see if that works. 

What's odd is I was able to run it on 3hr intervals with a script like this !11:00NmTPfWp.300!S!10800E!1R without the battery and it appeared to work fine.

**Reply by dnewman-gpsw** [reply]
Without a battery is fine for the first 24 hours or so.  This is coincell that recharges from the battery to preserve the RTC.  This can get weird after that, as it seems the coin cell doesn't recharge from USB (although I have research it this is true), I've only anecdotally seen more reliability issues without the battery.
